modules/emojifont/emojifont.js

```diff
@@ -0,0 +1,218 @@
+class Node {
+  constructor(values, children, parent) {
+    this.values = values;
+    this.children = children;
+    this.parent = parent;
+  }
+
+  isRoot() {
+    return this.parent === null;
+  }
+}
+
+function walkTree(node, key, depth, callback) {
+  callback(node, key, depth);
+  Object.keys(node.children).forEach(function(childKey){
+    walkTree(node.children[childKey], childKey, depth+1);
+  });
+}
+
+function printNodeTree(root) {
+  walkTree(root, "(ROOT)", 0, function (node, nodeKey, depth) {
+    const spacer = "--".repeat(depth) + ">";
+    console.log(spacer + " " + key +", Values: " + node.values);
+  });
+}
+
+function theWorksString() {
```

Comment:
more es6y is 

```suggestion
const theWorksString = () =>
```

Depending on scope of course